R/build-articles.R

```diff
@@ -47,8 +47,9 @@
 #'   \code{pkg} directory.
 #' @param depth Depth of path relative to root of documentation.
 #'   Used to adjust relative links in the navbar.
+#' @param encoding The encoding of the input files.
 #' @export
-build_articles <- function(pkg = ".", path = "docs/articles", depth = 1L) {
+build_articles <- function(pkg = ".", path = "docs/articles", depth = 1L, encoding = getOption("encoding")) {
```

Comment:
I think it's a bad idea to add a depedency on a global option because this means it might work on one computer but fail on another. I'd rather default to UTF-8, or the value of encoding in the DESCRIPTION